README.md

```
@@ -46,51 +48,97 @@ export default [
 ## Rules
 
 ### `yenz/type-ordering`
-Ensures that null/undefined types are listed last in TypeScript union types.
+
+Ensures that `null`/`undefined` types are listed last in TypeScript union types. Auto-fixable.
+
+Requires `typescript-eslint` parser to work (see [Installation](#installation)).
+
+**Bad:**
+```typescript
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)